### ISSUE TITLE
Absolute path for WORKDIR false positive on Windows

### ISSUE THREAD

**Issue by marcofranssen** [RedCoolBeans/dockerlint#3]
The absolute path rule for the work dir gives an error on Windows when the Dockerfile does contain an absolute path.

```
WORKDIR /data
```

On windows the check recognizes this path as relative where on Linux it works properly.

**Comment by jasperla** [comment]
Thank you for the report; I will fix this issue shortly.
